Packages/com.ramtype0.meshia.mesh-simplification/Ndmf/Runtime/OverallMeshiaMeshSimplifier.cs

```diff
@@ -0,0 +1,57 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Meshia.MeshSimplification.Ndmf
+{
+    public class OverallMeshiaMeshSimplifier : MonoBehaviour
+#if ENABLE_VRCHAT_BASE
+    , VRC.SDKBase.IEditorOnly
+#endif
+    {
+        public List<OverallMeshiaMeshSimplifierTarget> Targets = new();
+        public int TargetTriangleCount = 70000;
+        public bool IsAutoAdjust = false;
+    }
+
+    [Serializable]
+    public record OverallMeshiaMeshSimplifierTarget
+    {
+        public Renderer Renderer;
+        public int TargetTriangleCount;
+        public MeshSimplifierOptions Options;
+        public OverallMeshiaMeshSimplifierTargetState State;
+        public bool Fixed;
+
+        public OverallMeshiaMeshSimplifierTarget(Renderer renderer)
+        {
+            Renderer = renderer;
+            TargetTriangleCount = RendererUtility.GetMesh(renderer)!.triangles.Length / 3;
+            Options = MeshSimplifierOptions.Default;
+            State = OverallMeshiaMeshSimplifierTargetState.Enabled;
+            Fixed = false;
+        }
+
+        public static bool IsValidForTarget(Renderer renderer)
```

Comment:
```suggestion
        public static bool IsValidTarget(Renderer renderer)
```
現在の命名だとインスタンスメソッドと誤認しそうな気がする